kubel.el

```diff
@@ -664,19 +664,6 @@ TYPENAME is the resource type/name."
    (-contains? '("ReplicaSets" "replicasets" "replicasets.apps") kubel-resource)
    (-contains? '("StatefulSets" "statefulsets" "statefulsets.apps") kubel-resource)))
 
-(defun kubel--save-line ()
-  "Save the current line number if the view is unchanged."
-  (if (equal (buffer-name (current-buffer))
-             (kubel--buffer-name))
-      (setq kubel--line-number (+ 1 (count-lines 1 (point))))
-    (setq kubel--line-number nil)))
-
-(defun kubel--jump-back-to-line ()
-  "Jump back to the last cached line number."
-  (when kubel--line-number
```

Comment:
can we also delete `kubel--line-number` if it's not used?

Reply:
Done